direct/src/distributed/direct.dc

```diff
@@ -29,18 +28,21 @@ dclass DistributedObject {
   setLocation(uint32 parentId, uint32 zoneId) broadcast ram;
 };
 
+dclass DistributedObjectGlobal : DistributedObject {
+};
+
 dclass TimeManager: DistributedObject {
-  requestServerTime(uint8 context) p2p;
+  requestServerTime(uint8 context) aircv clsend;
```

Comment:
Typo of `airecv`

Also, `TimeManager` works in a very particular way with `ServerRepository`; while airecv/clsend is a more Astronic way of doing things, I don't know if this will break CMU's server (which I don't know if people use)